docs/proposals/admission_control_changes.md

```diff
@@ -0,0 +1,279 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/kubernetes/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Admission control changes
+
+We are proposing a series of changes to admission control in an attempt to make
+it less error-prone, easier to configure, and more flexible for admission plugin
+developers.
+
+
+## Hard-coded plugin execution order
+
+It's unlikely that admins need to change the order in which admission plugins
+are executed.  Furthermore, running certain plugins "out of order" could result
+in undesirable or incorrect behavior. For example, if you run `ResourceQuota`
+before `LimitRanger`, you could end up incorrectly calculating quota because
+it's possible for `LimitRanger` to modify pod values after `ResourceQuota`
+performed its quota checks.
+
+Because of this, we want to move to a model where admission plugin execution
+order is hard-coded. We would therefore deprecate the `--admission-control`
```

Comment:
say "compiled in admission controller is strongly ordered" and not under the administrator's control.